BootloaderCommonPkg/Library/BootloaderCommonLib/BootloaderCommonLib.c

```diff
@@ -390,6 +390,10 @@ GetComponentInfoByPartition (
   }
 
   FlashMapPtr = GetFlashMapPtr ();
+  if (FlashMapPtr == NULL) {
+    return EFI_NOT_FOUND;
+  }
+

```

Comment:
Is this required? GetComponentEntryByPartition() returns NULL if FlashMapPtr is NULL.
What about changing GetComponentEntryByPartition() API to get FlashMapPtr as a parameter?
FlashMapPtr = GetFlashMapPtr ();
if (FlashMapPtr == NULL) {
  return EFI_NOT_FOUND;
}
Entry = GetComponentEntryByPartition(FlashMapPtr, Signature, IsBackupPartition);
if (Entry == NULL) {
  return EFI_NOT_FOUND;
}

Reply:
I agree it is not a real issue.   But let us not change too much code here to make Klocwork happy.
If line 393 to 396 can resolve the klocwork issue,  how about we just go with it.

Reply:
Okay let's go with this if this change resolves KW report.